### PR TITLE
WLT-794: added saga in transfer

### PR DESCRIPTION
**Pull request by pol9kov**
**- What I did**
added saga in transfer
**- How I did it**
added saga in transfer
**- How to verify it**
launch test
**- Description for the changelog**

added saga in transfer

### CODE REVIEW

File: logicrunner/builtin/contract/wallet/wallet.go [hunk 1]
```diff
@@ -109,21 +109,16 @@ func (w *Wallet) Transfer(rootDomainRef insolar.Reference, amountStr string, toM
 		return member.TransferResponse{Fee: feeStr}, nil
```
Comment (bigbes) [inline logicrunner/builtin/contract/wallet/wallet.go]: Can you replace ok/error path? 
```if err != nil {
   return err
}
return ok```

File: logicrunner/builtin/contract/wallet/wallet.go [hunk 2]
```diff
@@ -147,8 +142,8 @@ func (w *Wallet) Accept(amountStr string) (err error) {
 	return nil
 }
 
-// RollBack rolls back transfer to balance.
-func (w *Wallet) RollBack(amountStr string) (err error) {
+// Rollback rolls back transfer to balance.
```
Comment (LinnikD) [inline logicrunner/builtin/contract/wallet/wallet.go]: Rollback is unused now? Do we need it?

Comment (LinnikD) [reply logicrunner/builtin/contract/wallet/wallet.go]: It ok, need it for saga calls

File: logicrunner/builtin/contract/wallet/wallet.go (hunk 1 above)
Comment (pol9kov) [reply logicrunner/builtin/contract/wallet/wallet.go]: Done

**Pull request merged**